.github/workflows/pre-commit-hook-run.yml

```diff
@@ -20,5 +20,5 @@ jobs:
               with:
                   node-version: "16"
             - run: git init && git add --all && git -c user.name='test' -c user.email='test@example.com' commit -m 'init for pr action'
-            - run: npm i
+            - run: npm i --force
```

Comment:
this change is already in branch 9.2. Also, since this is nothing to do with this PR, it should not be here in the first place.